wolfhsm/wh_packet.h

```diff
@@ -263,6 +263,37 @@ typedef struct  wh_Packet_cmac_res
     /* uint8_t out[]; */
 } wh_Packet_cmac_res;
 
+typedef struct wh_Packet_hash_any_req {
+    uint32_t type; /* enum wc_HashType */
+} wh_Packet_hash_any_req;
+
+typedef struct wh_Packet_hash_sha256_req {
+    /* TODO change to "wh_Packet_hash_any_req header" */
+    uint32_t type; /* enum wc_HashType */
+    struct {
+        /* Flag indicating whether to use the intermediate hash */
+        uint32_t resumeHashFlag;
+        uint32_t hiLen;
+        uint32_t loLen;
+        /* intermediate hash value */
+        uint8_t hash[32]; /* TODO (BRN) WC_SHA256_DIGEST_SIZE */
+    } resumeState;
+    /* Flag indicating to the server that this is the last block and it should
+     * finalize the hash. If set, inBlock may be only partially full*/
+    uint32_t isLastBlock;
+    /* Length of the last input block of data. Only valid if isLastBlock=1 */
+    uint32_t lastBlockLen;
+    /* Full sha256 input block to hash */
+    uint8_t inBlock[64]; /* TODO (BRN) WC_SHA256_BLOCK_SIZE */
```

Comment:
Why not use the WC_SHA256_BLOCK_SIZE?  Same of other todos

Reply:
Understood.  Recommend add additional #define's in here if you want to keep these sizes statically defined.